src/main/java/net/minecraftforge/client/CloudRenderer.java

```diff
@@ -0,0 +1,417 @@
+package net.minecraftforge.client;
```

Comment:
Please add the missing license header, just copy it from any other forge code file.